app/src/main/resources/application.properties

```diff
@@ -119,8 +119,8 @@ mp.openapi.servers=/api
 %dev.registry.enable.multitenancy=false
 %prod.registry.enable.multitenancy=false
 
-quarkus.kafka.devservices.enabled=false
-quarkus.apicurio-registry.devservices.enabled=false
+%test.quarkus.kafka.devservices.enabled=false
```

Comment:
This will make devservices start in dev mode, right? Have you tested this change and confirmed that the application is still running fine with mvn quarkus:dev?

Reply:
no, haven't tested

Reply:
If someone can verify that `mvn quarkus:dev` still works OK with this change then it's cool...

Reply:
It works. it's starting a kafka redpanda devservice even if it's not needed, but it's working.

Reply:
I'll fix this, to include these properties in dev profile as well, I just don't want it in the prod profile